.github/CODEOWNERS

```diff
@@ -165,6 +165,10 @@ datadog_checks_base/tests/**/test_db_statements.py  @DataDog/database-monitoring
 /oracle/*.md                                        @DataDog/agent-integrations @DataDog/database-monitoring @DataDog/documentation
 /oracle/manifest.json                               @DataDog/agent-integrations @DataDog/database-monitoring @DataDog/documentation
 
+# APM Integrations
+/langchain/         @DataDog/apm-core-python @DataDog/llm-obs
+/openai/            @DataDog/apm-core-python @DataDog/llm-obs
```

Comment:
Could you also keep the @DataDog/agent-integrations and @DataDog/documentation teams in the loop similar to what we have for other integrations that we do not primarily own?